src/app/services/communibee-backend/backend-model.ts

```diff
@@ -0,0 +1,4 @@
+export default interface BackendModel {
```

Comment:
Why is this interface needed?
Does it really have to contain `_id` an `_v` members?

Reply:
yes, because of the way that you work with mongoose models in beckend, and the _id is the id of the model....